### PR TITLE
DOCSP-21519: node TLDR

### PR DESCRIPTION
**Pull request by ccho-mongodb**
## Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-21519

### Snooty build log:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=624ded910fdfea9c37ff2e74

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-21519-node-tldr/quick-reference/

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging and workerpool job links in the PR description updated?

### If your page documents a concept, does it meet the following criteria?

- [x] Target the [Jasmin persona](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view)
- [x] Target the [Lucas persona](https://drive.google.com/file/d/1J2vqJxo7ldv7OP_obA9Q-avf0o_ju4Lk/view)

### CODE REVIEW

File: source/usage-examples/updateOne.txt [hunk 1]
```diff
@@ -8,7 +10,7 @@ Update a Document
    If you specify a callback method, ``updateOne()`` returns nothing. If you
    do not specify one, this method returns a ``Promise`` that resolves to the
    result object when it completes. See our guide on :doc:`Promises and
-   Callbacks </fundamentals/promises>` for more information, or see the
+   Callbacks </fundamentals/promises>` for more information, or see the˛˛
```
Comment (terakilobyte) [inline source/usage-examples/updateOne.txt:13]: issue: I'm unsure what this punctuation is supposed to represent.

**Review by terakilobyte — CHANGES_REQUESTED** [review]
Really great work here! Just a few minor issues I noted.

File: source/quick-reference.txt [hunk 2]
```diff
@@ -0,0 +1,455 @@
...
+
+     - .. io-code-block::
+          :copyable: true
+
+          .. input::
+             :language: js
+
+             await coll.updateOne(
+                { title: 'Amadeus' },
+               { $set: { 'imdb.rating': 9.5 } }
```
[the remark this answers is not in the document]
Comment (ccho-mongodb) [reply source/quick-reference.txt:100]: Whoops, looks like VSCode uses 3-space indents. I wanted 2-space. Fixing all instances to 2-space.

File: source/quick-reference.txt [hunk 3]
```diff
@@ -0,0 +1,455 @@
...
+                   document: {
+                     title: 'A New Movie',
+                     year: 2022
+                   }
+               },
+                 deleteMany: {
+                   filter: { year: { $lt: 1970 } }
+                 }
+               }
+             ]);
```
Comment (terakilobyte) [inline source/quick-reference.txt:222]: issue: I may have missed this in my previous review, but the indentation/brackets seem off here?

```suggestion
               {
                 insertOne: {
                   document: {
                     title: 'A New Movie',
                     year: 2022
                   }
                 }
               },
               {
                 deleteMany: {
                   filter: { year: { $lt: 1970 } }
                 }
               }
             ]);
```

**Review by terakilobyte — APPROVED** [review]
LGTM with one correction or explanation to the bulkwrite example shown.

File: source/quick-reference.txt (hunk 3 above)
Comment (ccho-mongodb) [reply source/quick-reference.txt:222]: Good catch

**Review by kyuan-mongodb — APPROVED** [review]
LGTM!

**Pull request merged**